src/Template/Bake/Model/entity.ctp

```diff
@@ -13,6 +13,13 @@
  * @license       http://www.opensource.org/licenses/mit-license.php MIT License
  */
 
+use Cake\ORM\Association;
+
+$propertyHintMap = null;
```

Comment:
Ok, then I'll leave it like that for now, just for the sake of having less to consider when creating tests, and for cleaner looking comparison files :)